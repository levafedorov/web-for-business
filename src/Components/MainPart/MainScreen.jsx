import React from 'react';
import MainPartMessage from './MainPartMessage';
import Positioner from '../Page/Positioner';


export default function MainScreen(props){
   
   
        
    
        return <div className="main-screen">
                 {props.children}
                 <div className="text-place">
                 <Positioner>
                <MainPartMessage />
                 </Positioner></div>
               </div>
    
}