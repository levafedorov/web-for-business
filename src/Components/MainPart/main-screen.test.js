import React from 'react';
import MainScreen from './MainScreen';
import {shallow, mount} from 'enzyme';







describe('CompactSlidingMenuTest component', () => {
    const setUp = (props) => shallow(<MainScreen {...props}/>);
    let component;
    let instance;
    beforeEach(() => {
        component = setUp();
        instance = component.instance();
    });

    it("Should component render", () => {
        component = setUp();
        expect(component).toMatchSnapshot();
    });

//     describe('Component methods', () => {
      
            
// });

});
