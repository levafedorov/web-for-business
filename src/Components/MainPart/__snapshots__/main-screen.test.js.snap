// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CompactSlidingMenuTest component Should component render 1`] = `
<div
  className="main-screen"
>
  <div
    className="text-place"
  >
    <Positioner>
      <MainScreen />
    </Positioner>
  </div>
</div>
`;
